### ISSUE TITLE
Failing Color Contrast Ratio

### ISSUE THREAD

**Issue by brandonjpierce** [bpxl-labs/usoniancss#4]
[Link - Accessibility test results](http://jxnblk.com/colorable/demos/text/?background=%23f5f5f5&foreground=%2363e7ca)
[Button Component - Accessibility test results](http://jxnblk.com/colorable/demos/text/?background=%2363e7ca&foreground=%23ffffff)

Button and anchor tag contrast ratios [fail WCAG standard](https://www.w3.org/TR/WCAG20/#visual-audio-contrast).

Instead of supplying a predetermined color scheme in Usonian, something like [mrmrs/colors](https://github.com/mrmrs/colors) could be added instead which has a list of [accessible colors combos](http://clrs.cc/a11y/).
